README.md

```diff
@@ -24,30 +24,30 @@ that reads from the feed.
  */
 
 //create a secret-stack instance and add ssb-db, for persistence.
-var createSbot = require('secret-stack')({})
+var createSsbServerWithDb = require('secret-stack')({})
```

Comment:
Sorry to nit, but I think this might give the impression that it's creating an ssb-server instance.

Really, it's making an instance of secret-stack, which we're loading the ssb-db plugin into. For consistency with the [secret-stack docs](https://github.com/ssbc/secret-stack/#example) maybe this should be `createApp()`?

There's also some other weird context where most secret-stack plugins namespace their methods, but this plugin is naughty and makes top-level methods. In a consistent universe we'd do something like:

```javascript
const createApp = require('secret-stack')({})
  .use('ssb-db')

const app = createApp(require('ssb-config')

app.db.createHistoryStream()
//  ^ note the `db` here, which we don't actually do in the real world because we're inconsistent
```

Reply:
ah, true. No, these distinctions are important, I'm into it.

Reply:
If you're down with this change I'm happy to make the edit so you don't have to add another commit, but I wanted to make sure I discussed with you because you have a much better perspective on the documentation. Working in an ecosystem for too long warps your brain and it makes everything feel "normal", which is often really bad for writing good documentation.